### PR TITLE
ADEN-7012 Adding rest of WebAds AppNexus placements

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js [hunk 1]
```diff
@@ -15,15 +15,67 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusWebAds',[
...
+					sizes: [
+						[728, 90],
+						[970, 250]
+					]
+				},
+				/* WIKIA_FMR_AS */
 				INCONTENT_BOXAD_1: {
 					placementId: '12939349',
 					sizes: [
-						[300, 600],
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js:21]: I know it wasn't listed in file, but is it ok that we remove those sizes?

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js [hunk 2]
```diff
@@ -15,15 +15,67 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusWebAds',[
...
+					sizes: [
+						[320, 50]
+					]
+				},
+				/* WIKIA_mIC_AS */
+				MOBILE_IN_CONTENT: {
+					placementId: '13104397',
+					sizes: [
+						[300, 250],
+						[320, 240],
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js]: We don't support that size. Is it going to work?

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js [hunk 3]
```diff
@@ -15,15 +15,67 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusWebAds',[
...
+						[320, 100],
+						[320, 50]
+					]
+				},
+				/* WIKIA_BLB_AS */
+				BOTTOM_LEADERBOARD: {
+					placementId: '13104398',
+					sizes: [
+						[300, 250],
+						[320, 240],
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js]: We don't support that size. Is it going to work?

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js]: Removing.

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js (hunk 2 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js]: Removing.

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js:21]: Leaving removed.

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**